app/templates/events/trackVolunteers.html

```diff
@@ -248,43 +246,29 @@ <h5 class="modal-title" id="exampleModalLabel">{{modalTitle}}</h5>
             <i class="bi bi-search ms-0"></i>
           </button>
         </div>
-        <br>
-        <h5>Selected Volunteers</h5>
-        <ul class="list-unstyled" id= "addVolunteerList">
+        <ul class="list-unstyled mx-3 mt-2" id= "addVolunteerList">
+          {% if event.isRecurring and not event.isFirstRecurringEvent%}
+            {% if recurringVolunteers | length != 0  %}
+              {%for participant in recurringVolunteers%}     
+                <li>
+                  <input class="recurringVolunteer" type="checkbox" id ="recurringVolunteerCheckbox_{{loop.index}}" name="volunteer[]" value= {{participant}} />
+                  <label class="form-check-label" for="recurringVolunteerCheckbox_{{loop.index}}"> {{participant.firstName}} {{participant.lastName}}</label>
+                  <label class="text-dark" data-toggle="previousVolunteerHover">*</label>
+                </li>
+              {% endfor %}
+            {% endif %}
+          {% endif %}
         </ul>
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-secondary me-auto" data-bs-dismiss="modal">Close</button>
+        {% if event.isRecurring and (recurringVolunteers | length != 0) and not event.isFirstRecurringEvent  %}
```

Comment:
You can probably use `recurringVolunteers` on its own rather than using length and comparing to 0, as an empty string evaluates to False. I'd check that in jinja first. And if not, adding `| length` without the comparison should work.

Also, `not event.isFirstRecurringEvent` is kind of redundant, as `recurringVolunteers` will always be empty in that case, so we'll shortcut the expression before we get to the end.

Reply:
Thanks Bryon for the awesome conditional tip!

About the redundancy in the `not event.isFirstRecurringEvent`, that part of the conditional is there to make sure that volunteers that have been added to the first of a series of events are not suggested as recurring volunteers for that same instance of the event (that is, week 1). I believe that we should leave this one in.